build.sh

```diff
@@ -703,4 +703,8 @@ if [[ -n "${DORIS_POST_BUILD_HOOK}" ]]; then
     eval "${DORIS_POST_BUILD_HOOK}"
 fi
 
+if [[ -f "${CMAKE_BUILD_DIR}/compile_commands.json" ]]; then
+    ln -s "${CMAKE_BUILD_DIR}/compile_commands.json" "${DORIS_HOME}/compile_commands.json"
+fi
+
```

Comment:
I don't think the path `${DORIS_HOME}/compile_commands.json` is appropriate. We have two build scripts, one is for `src` and another one is for `test`. These build scripts will generate different `compile_commands.json`. Linking the `compile_commands.json` to the root of the project would affects these build targets.